spec/factories/enterprise_factory.rb

```diff
@@ -14,6 +14,7 @@
     description { 'enterprise' }
     long_description { '<p>Hello, world!</p><p>This is a paragraph.</p>' }
     address
+    visible { 'public' }
```

Comment:
I guess this is for backwards-compatibility with specs assuming this value? 
If so, I think it would have been better to specify it in the specs themselves (they are now using a context with a non-default value). Perhaps the specs might need updating to test the default value too.

Reply:
> I guess this is for backwards-compatibility with specs assuming this value?

Yes. 

> If so, I think it would have been better to specify it in the specs themselves (they are now using a context with a non-default value).

I had hesitation yes ... I chose the easiest solution, but maybe not the best one... I will change that if you think it's better. Maybe will create a new trait for that.